_scripts/fetch-upstream-resources.sh

```diff
@@ -16,10 +17,15 @@ distribution_svn_branch="branches/${DISTRIBUTION_BRANCH}"
 if [ "${distribution_svn_branch}" = "branches/master" ]; then
 	distribution_svn_branch=trunk
 fi
+compose-cli_svn_branch="branches/main"
```

Comment:
This should contain the `COMPOSE_CLI_BRANCH` env-var;

```suggestion
compose-cli_svn_branch="branches/${COMPOSE_CLI_BRANCH}"
```

Reply:
fixed